src/reducers/index.ts

```diff
@@ -18,9 +42,84 @@ function reducer(state: Readonly<StateBase>, action: Action): StateBase {
   };
 }
 
+/**
+ * Exclude these actions from the history completely.
+ */
+export const ACTIONS_EXCLUDED_FROM_HISTORY: ActionType[] = [
```

Comment:
Hmm. I think they are declarations of different categories of actions.   The purpose is for history, but whenever you work on new action, you have to add each action to one of these arrays.